### PR TITLE
MD-05 into develop

### PR DESCRIPTION
**Pull request by DaanEKR**
# What does this implement/fix? Explain your changes.
Setup general styling;

- Created input mixin
- Created input checkbox component
- added Normalize

## Does this close any currently open issues?
#6 

## Where has this been tested?
**Browsers:**
- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Edge

## Needs 2 thumbs up

### CODE REVIEW

**Comment by koenkort** [comment]
Duimpie

File: front-dex/src/app/components/caught/caught.component.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="checkbox">
+    <input id="check1" type="checkbox">
```
Comment (Danny-Mus) [inline front-dex/src/app/components/caught/caught.component.html]: check1?

File: front-dex/src/app/components/favourite/favourite.component.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="checkbox">
+    <input id="check" type="checkbox">
```
Comment (Danny-Mus) [inline front-dex/src/app/components/favourite/favourite.component.html]: check? naming?

File: front-dex/src/index.html [hunk 3]
```diff
@@ -8,6 +8,7 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 </head>
 <body>
-  <app-root></app-root>
+<input type="text" placeholder="Find Pokémon" class="input">
```
Comment (Danny-Mus) [inline front-dex/src/index.html]: remove input

File: front-dex/src/styles.scss [hunk 4]
```diff
@@ -1 +1,5 @@
-/* You can add global styles to this file, and also import other style files */
+@import './assets/scss/main';
+
+.input {
```
Comment (Danny-Mus) [inline front-dex/src/styles.scss]: remove `input` styling

**Pull request merged**